<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationWidget</class>
 <widget class="QWidget" name="NavigationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>137</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navigation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="m_splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="locationListView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="toolTip">
          <string>Location Browser</string>
         </property>
         <property name="whatsThis">
          <string>Double click an entry of this list to get it displayed on the map.</string>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="layoutMode">
          <enum>QListView::Batched</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="_3">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="5" column="1">
          <widget class="QToolButton" name="zoomOutButton">
           <property name="toolTip">
            <string>Zoom Out</string>
           </property>
           <property name="whatsThis">
            <string>Zoom out by pressing this button to see less detail.</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../marble.qrc">
             <normaloff>:/icons/zoom-out.png</normaloff>:/icons/zoom-out.png</iconset>
           </property>
           <property name="shortcut">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="goHomeButton">
           <property name="toolTip">
            <string>Navigate to your 'Home Location'</string>
           </property>
           <property name="whatsThis">
            <string>Click this button to restore the original zoom level and view angle.</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../marble.qrc">
             <normaloff>:/icons/go-home.png</normaloff>:/icons/go-home.png</iconset>
           </property>
           <property name="shortcut">
            <string>Home</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="moveRightButton">
           <property name="toolTip">
            <string>Right</string>
           </property>
           <property name="whatsThis">
            <string>Rotates the globe clockwise around its axis.</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../marble.qrc">
             <normaloff>:/icons/arrow-right.png</normaloff>:/icons/arrow-right.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="moveUpButton">
           <property name="toolTip">
            <string>Up</string>
           </property>
           <property name="whatsThis">
            <string>Tilts the globe's axis towards the user.</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../marble.qrc">
             <normaloff>:/icons/arrow-up.png</normaloff>:/icons/arrow-up.png</iconset>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="zoomSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Zoom Slider</string>
           </property>
           <property name="whatsThis">
            <string>Use this slider to adjust the zoom level of the map.</string>
           </property>
           <property name="minimum">
            <number>900</number>
           </property>
           <property name="maximum">
            <number>2400</number>
           </property>
           <property name="value">
            <number>1050</number>
           </property>
           <property name="sliderPosition">
            <number>1050</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="zoomInButton">
           <property name="toolTip">
            <string>Zoom In</string>
           </property>
           <property name="whatsThis">
            <string>Zoom in by pressing this button to see more detail.</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../marble.qrc">
             <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
           </property>
           <property name="shortcut">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="moveDownButton">
           <property name="toolTip">
            <string>Down</string>
           </property>
           <property name="whatsThis">
            <string>Tilts the globe's axis away from the user.</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../marble.qrc">
             <normaloff>:/icons/arrow-down.png</normaloff>:/icons/arrow-down.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="moveLeftButton">
           <property name="toolTip">
            <string>Left</string>
           </property>
           <property name="whatsThis">
            <string>Rotates the globe counterclockwise around its axis.</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../marble.qrc">
             <normaloff>:/icons/arrow-left.png</normaloff>:/icons/arrow-left.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_pSpacerFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../marble.qrc"/>
 </resources>
 <connections/>
</ui>
